sys/include/can/can.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    can
+ * @{
+ *
+ *
+ * @file
+ * @brief       Definitions high-level CAN interface
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @author      Toon Stegen <toon.stegen@altran.com>
+ */
+
+#ifndef RIOT_CAN_H
+#define RIOT_CAN_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+#if defined(__linux__)
+
+#include <linux/can.h>
+#include <libsocketcan.h>
+
+#else
+
+#define CAN_MAX_DLEN 8
```

Comment:
I know I'm a bit manic in this regard, but I always prefer to have macro definitions with parens (or at least consistent throughout a module.)